api/libproject/devstart.rb

```diff
@@ -194,8 +194,8 @@ def run_local_migrations()
     common.run_inline %W{./run-migrations.sh main}
   end
   Dir.chdir('db-cdr/generate-cdr') do
-    common.run_inline %W{./init-new-cdr-db.sh --cdr-db-name cdr}
-    common.run_inline %W{./init-new-cdr-db.sh --cdr-db-name public}
+    common.run_inline %W{./init-new-cdr-db.sh --cdr-db-name cdr --version-flag cdr}
```

Comment:
Also please update all the calls that hit this function: https://github.com/all-of-us/workbench/blob/72b5eafff9700cc4587094597bdd5705d19fbffb/api/libproject/devstart.rb#L88

Reply:
Updated. Tested with local data migrations. Merging in.